xml/chapter1/section2/subsection6.xml

```diff
@@ -683,6 +683,31 @@ timed_prime_test(43);
     prediction?  Is your result compatible with the notion that programs on
     your machine run in time proportional to the number of steps required for
     the computation?
+    <SOLUTION>
+      <SNIPPET>
+        <NAME>search_for_primes_solution</NAME>
+        <REQUIRES>timed_prime_definition</REQUIRES>
+        <JAVASCRIPT>
+          function search_for_primes(start, times) {
+              return times === 0
+                  ? true
+                  : start > 2 && start % 2 === 0
```

Comment:
You need to replace
&&
by
&amp;&amp;
to comply with XML syntax. Also in the same line:
>
should be
&gt;